website/docs/icons/usage-guidelines/partials/guidelines/overview.md

```diff
@@ -62,6 +62,16 @@ If an object can have multiple states, use a Contained icon for the overall pare
 
 ![Example of contained icons in context](/assets/foundations/flight-icons/icon-best-practices-contained.png =660x*)
 
+### Animated
+
+Animated icons are icons with an animated effect that show a transition between two states. 
+
+![Example of animated icons](/assets/foundations/flight-icons/icon-animated.png =660x*)
```

Comment:
I saw a comment by @heatherlarsen about having GIFs for these images (but I may have misunderstood). Can you double check with her?

Reply:
In the original [issue](https://github.com/hashicorp/design-system/issues/27), I provided animated GIFs, but those won't work here, as they have different sizes and styles. Since we don't have a component where we can render the actual icons, I think it's okay to have the static reference for now.